src/helpers/money.php

```diff
@@ -1,10 +1,10 @@
 <?php
 
-function money($input, $showCents = true)
+function money($input, $showCents = true, $locale = null)
 {
-    setlocale(LC_MONETARY, locale_get_default());
+    setlocale(LC_MONETARY, $locale ? $locale : locale_get_default());
```

Comment:
As 7.1 is the minimum PHP version, you could use `$locale ?? locale_get_default()`

Reply:
Sorry did the change quite late - you are right. I had forgotten the min 7.1 requirement.